services/treeware/treeware-trees.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { isMetric } = require('../test-validators')
+
+t.create('request for existing package')
+  .timeout(10000)
+  .get('/stoplightio/spectral.json')
+  .expectBadge({
+    label: 'trees',
+    message: isMetric,
+  })
+
+t.create('request for existing profile (mock)')
+  .get('/stoplightio/spectral.json')
+  .intercept(nock =>
+    nock('https://public.offset.earth')
+      .get('/users/treeware/trees?ref=65c6e3e942e7464b4591e0c8b70d11d5')
+      .reply(200, { total: 50 })
+  )
+  .expectBadge({
```

Comment:
If this test is mocked, you should add the exact count to the expectation as well as the colour - otherwise there is little added value compared to the test that hits the real API.